content/200-concepts/100-components/02-prisma-client/080-null-and-undefined.mdx

```diff
@@ -134,3 +134,158 @@ updateUser: (parent, args, ctx: Context) => {
   })
 },
 ```
+
+## The effect of <inlinecode>null</inlinecode> and <inlinecode>undefined</inlinecode> on conditionals
+
+There are some caveats to filtering with conditionals which might produce unexpected results. When filtering with conditionals you might expect one result but receive another given how Prisma treats `nullable` values.
+
+### A high level overview
+
+
+|Operator                     | 0 filters              | 1 filter               | n filters            |
+|-----------------------------|------------------------|------------------------|----------------------|
+|<inlinecode>OR</inlinecode>  | return empty list      | validate single filter | validate all filters |
+|<inlinecode>AND</inlinecode> | return all items       | validate single filter | validate all filters |
+|<inlinecode>NOT</inlinecode> | return all items       | validate single filter | validate all filters |
```

Comment:
```suggestion
|`NOT` | return all items       | validate single filter | validate all filters |
```

Reply:
The `inlinecode` thing is only required for code in headings, because it's needed to render the code style in navigation elements (like right-hand TOC) - I've made myself a task to document it because it hasn't been! https://github.com/prisma/docs/issues/1440